src/app-feature/account-feature/pages/login-page/login-page.component.ts

```diff
@@ -51,13 +52,14 @@ export class LoginPageComponent implements OnInit {
     }
 
     const credentials: IAccountCredentials = this.loginForm.getRawValue();
-    const onSuccess = (): void => {
-      const notificationConfig: INotificationConfiguration = {
-        message: 'TODO: Implement user authentication feature.',
-      };
-      this.context.ui.showNotification(notificationConfig);
-    };
+    this.accountService.login(credentials).subscribe(this.onLoginSuccess.bind(this));
```

Comment:
Why not keep it as a local function? I think that we should almost always have local functions, and then, for situations where there is a lot of code, we could move some of that code to a private method. But as a rule, we should have localy defined functions.